### PR TITLE
Deceased journey content fixes

### PR DESCRIPTION
**Pull request by stephenjmcneill1971**
Deceased journey content fixes

### CODE REVIEW

File: src/lib/resource/sections/applicant-contact-out-of-touch.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
                                 proxy: 'This helps us to understand their eligibility.'
                             },
                             error: {
-                                myself: 'Tell us why you not in contact',
+                                myself: 'Tell us why you were not in contact',
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-contact-out-of-touch.js:25]: Why does this one not have a full stop and proxy does?

File: src/lib/resource/sections/applicant-fatal-claim.js [hunk 2]
```diff
@@ -13,13 +13,13 @@ module.exports = {
                     title: 'Select why you are applying',
                     oneOf: [
                         {
-                            title: 'I am applying because of an injury',
+                            title: 'I am applying because of an injury.',
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-fatal-claim.js:16]: Full stops have been added to all of these properties but titles and descriptions in other pages on this PR do not?

File: src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js [hunk 3]
```diff
@@ -5,22 +5,20 @@ module.exports = {
...
                     resources: {
-                        'q-applicant-applied-for-other-compensation-briefly-explain-why-not': {
-                            title:
-                                'Tell us why you have not applied for or received any other compensation',
-                            'title_someone-else':
-                                'Tell us why no application has been made or payment received for any other form of compensation'
+                        'applicant-applied-for-other-compensation-briefly-explain-why-not': {
+                            title: {
+                                myself:
+                                    'Tell us why you have not applied for or received any other compensation or damages',
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js]: No punctuation

File: src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js [hunk 4]
```diff
@@ -5,22 +5,20 @@ module.exports = {
...
+                                proxy:
+                                    'Tell us why no application has been made or payment received for any other form of compensation or damages'
```
Comment (neil-stephen-mcgonigle) [inline src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js]: No punctuation

**Review by neil-stephen-mcgonigle — CHANGES_REQUESTED** [review]

File: src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js (hunk 3 above)
Comment (stephenjmcneill1971) [reply src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js]: Question marks are being left of sentences to make them less investigative.

File: src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js (hunk 4 above)
Comment (stephenjmcneill1971) [reply src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js]: Question marks are being left of sentences to make them less investigative.

File: src/lib/resource/sections/applicant-fatal-claim.js (hunk 2 above)
Comment (stephenjmcneill1971) [reply src/lib/resource/sections/applicant-fatal-claim.js]: The files in the PR follow these rules: Titles have no full stops unless question marks are needed or wanted. Descriptions have full stops. Error messages have full stops only if they have more than one sentence.

**Pull request merged**